Dockerfile

```diff
@@ -0,0 +1,33 @@
+# A base Docker image running Python 3.6.2 and Debian 8 "Jessie"
+# See: https://github.com/docker-library/python/blob/master/3.6/jessie/Dockerfile
+FROM python:3.6.2-jessie
+
+RUN apt-get update && \
+    apt-get install -y \
+    libboost-system-dev libboost-filesystem-dev libboost-chrono-dev \
+    libboost-program-options-dev libboost-test-dev libboost-thread-dev \
+    build-essential libtool autotools-dev automake pkg-config libssl-dev \
+    libevent-dev bsdmainutils \
+    libzmq3 miniupnpc
+
+# Install clang for potential use over g++.
+RUN apt-get install -y clang
+
+# The Bitcoin source code will be located at /code and mounted by 
+# docker-compose.
+ENV HOME /code
+WORKDIR /code
+
+# Install Berkeley DB 4.8 for use with wallet functionality.
+ENV BDB_PREFIX /db4
+RUN mkdir -p $BDB_PREFIX
+RUN wget 'http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz'
```

Comment:
1. it better to use https link with --no-check-certificate option (until download.oracle.com have valid SSL cert)
2. it better to use -O parameter to specify filename of downloaded file (there was issues in filename auto-detection of wget in the past)
3. use environment variable for 'db-4.8.30.NC' string and use it instead below (can be updated easily in future)